experiments/modelfree/finetuning_defense.py

```diff
@@ -0,0 +1,216 @@
+import os.path as osp
```

Comment:
Is this file still used anywhere?

Reply:
We can remove it, it was mostly just a helper file for running quick tests outside of the multi framework